### ISSUE TITLE
Mean, std normalization inconsistent

### ISSUE THREAD

**Issue by aabbas90** [tensorflow/models#9100]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [y] I am using the latest TensorFlow Model Garden release and TensorFlow 2.
- [y] I am reporting the issue to the correct repository. (Model Garden official or research directory)
- [y] I checked to make sure that this issue has not been filed already.

## 1. The entire URL of the file to debug:
https://github.com/tensorflow/models/blob/master/research/object_detection/model_lib_v2.py

## 2. Describe the bug:
The mean and std normalization of COCO images after preprocessing is still not in normal range in CenterNet. On `efficientdet_d0_coco17_tpu-32` behaviour is normal and preprocessing is giving close to 0 mean and 1 std. 
I have tried two ways to experiment with CenterNet by using `centernet_resnet_v1_50_fpn_512x512_coco17` config from the model zoo (without any checkpoint):

**1. By mentioning means and std in config:**
Means and stds taken as is from pre-trained model config file in the model zoo:
```
channel_means: [104.01362025, 114.03422265, 119.9165958 ]
channel_stds: [73.6027665 , 69.89082075, 70.9150767 ]
```
The mean and std of images _after_ normalization are computed from: https://github.com/tensorflow/models/blob/3f6fe2aa410d901aae8829597a65d084bffc20d3/research/object_detection/model_lib_v2.py#L598

```
features['image'].numpy().mean()
-115.607056
features['image'].numpy().std()
8.1939125
```
Notice the very negative value of mean and large value of std (batch size: 64).

**2. By mean:0, std:1 or not mentioning mean, std in config:**
```
channel_means: [0, 0, 0]
channel_stds: [1, 1, 1]
```

```
features['image'].numpy().mean()
-54.237076
features['image'].numpy().std()
80.218575
```

**3. If using `efficientdet_d0_coco17_tpu-32` not containing any mean, std info:**

```
features['image'].numpy().mean()
-0.9214938
features['image'].numpy().std()
1.362439
```

## 3. Steps to reproduce

Run https://github.com/tensorflow/models/blob/master/research/object_detection/model_main_tf2.py on config file of `centernet_resnet_v1_50_fpn_512x512_coco17` from TF model zoo and coco dataset. 

## 4. Expected behavior

Mean and std normalization should be consistent and inner workings documented. After standardization mean and std should be 0 and 1 respectively.

## 6. System information

- OS Platform and Distribution (e.g., Linux Ubuntu 18.04): 18.04
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): 2.3.0
- Python version: 3.7

**Comment by aabbas90** [comment]
@vighneshbirodkar Can you please comment on CenterNet behaviour of preprocessing? Thanks

**Comment by vighneshbirodkar** [comment]
@aabbas90 Can you share your config that you used to get those means/stds ? Are you trying to use the means/stds used in hourglasss with a ResNet backbone ? That is not the right thing to do, since the ResNet feature extractor does normalizing on its own
https://github.com/tensorflow/models/blob/master/research/object_detection/models/center_net_resnet_feature_extractor.py#L99

If there are some more clarifications needed, feel free to re-open.

**Comment by aabbas90** [comment]
Hi,

Attached is the exact config file I am using. This one, as you can see, does not contain any mean and std information, therefore the backed-in resnet normalization should kick-in. The final mean and std values of the preprocessed data I get is:

`features['image'].numpy().mean()
-55.525608
features['image'].numpy().std()
77.819305`

Which is still not close to 0 mean and 1 std.

[centernet_resnet_v1_50_fpn_512x512_coco17.zip](https://github.com/tensorflow/models/files/5073341/centernet_resnet_v1_50_fpn_512x512_coco17.zip)

**Comment by vighneshbirodkar** [comment]
You are seeing that because the resnet preprocessed inputs using `caffe` mode
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/applications/resnet.py#L524
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/applications/imagenet_utils.py#L73

This specifically does not do any rescaling, that's why you observe large values. With that said. the ResNet v1 network which follows later is trained using the same setting and should not have any problems with that input.

**Comment by aviraljain99** [comment]
Hi @vighneshbirodkar

I am trying to use [SSD MobileNet V2 FPNLite 640x640](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/tf2_detection_zoo.md) from the TF2 Object Detection Model zoo and convert it to tflite. I have trained this model on a custom dataset and I am trying to use post-training quantization. 

Could you guide me as to how I can find out the correct way to standardize the input to this model which I am specifically interested in? Where can I find what pre-processing is done by the model (or the training process) on the input images? I am also looking into using [SSD ResNet50 V1 FPN 640x640 (RetinaNet50)](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/tf2_detection_zoo.md) for my task and I am interested in knowing how the images for this model are pre-processed during training. This will be immensely helpful since at present I am getting poor predictions from my model and I believe I am passing inputs to the model differently to how the model was trained.

Thanks,
